rs080n_moveit_config/CMakeLists.txt

```diff
@@ -8,3 +8,4 @@ catkin_package()
 install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
   PATTERN "setup_assistant.launch" EXCLUDE)
 install(DIRECTORY config DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install(DIRECTORY script DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
`rs080n_moveit_config` doesn't have a script directory.
So this is not needed.

Reply:
Unfortunate copy-pasta (I assumed all moveit configs had the example script).